doc/developer_guide/launch.vs.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "0.2.1",
+  "defaults": {},
+  "configurations": [
+    {
+      "type": "default",
+      "project": "CMakeLists.txt",
+      "projectTarget": "cadet-cli.exe (Install) (bin\\cadet-cli.exe)",
```

Comment:
Backslash and `.exe` are both specific to Windows, please ensure this also works for Linux / MacOS.

Reply:
The launch.vs.json is for Visual Studio, so it's Windows specific anyways.